.syncpackrc

```diff
@@ -245,6 +245,15 @@
 			],
 			"pinVersion": "^2.3.2"
 		},
+		{
+			"dependencies": [
+				"glob"
+			],
+			"packages": [
```

Comment:
👋 FYI in case you folks prefer it, `"packages": ["**"]` can be omitted and will apply to every package by default. It can be nice to leave it in as you have been doing as it helps hint where it applies to, but they can also be cleaned out of this file if you preferred it that way.

Reply:
@JamieMason Thanks for the tip.

Reply:
I tested removing one of those and `fix-mismatches` logged a lot of errors. I think we'll need to update syncpack before that option is available.

Reply:
ahh my bad, I hadn't checked your version – ignore me 👍